setup.py

```diff
@@ -5,38 +5,41 @@
 except ImportError:
     from distutils.core import setup
 
-
 install_reqs = ['cached-property',
```

Comment:
not sure, python 3 already comes with statistics.
maybe a note in the intall doc ?

On 21 March 2016 at 16:33, mrname <notifications@github.com> wrote:

> In setup.py
> <https://github.com/mrname/haralyzer/pull/12#discussion_r56853149>:
>
> > @@ -5,38 +5,41 @@
> >  except ImportError:
> >      from distutils.core import setup
> >
> > -
> >  install_reqs = ['cached-property',
>
> Can we add backports.statistics here? Pretty sure it should be
> installable for any version/system. Still good to have the manual fallback
> that you added, but it would be nice to force the package to install this I
> would think.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mrname/haralyzer/pull/12/files/f836c874548301869ca24cbc475eb3b8c28df988#r56853149>
>